css/style.css

```diff
@@ -8,11 +8,39 @@ html {
 }
 
 body {
-  font-family: playfair_displayregular, Helvetica, sans-serif, Poppin;
+  font-family: 'playfair_displayregular', Helvetica, sans-serif, Poppin;
   margin: 0;
   background-color: white;
 }
 
+textarea {
+  min-height: 112px;
+  border-radius: 4px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 21px;
+  color: #7d7c84;
+  font-family: 'poppinsregular', Helvetica, sans-serif;
+  padding: 2% 2%;
+}
+
+footer {
+  display: flex;
```

Comment:
You have used display: flex 14 times. It is not a good practice to use repetitive CSS. I would advice you to create a class with a name `d-flex` and use it whenever you need it